src/HtmlEntities.php

```diff
@@ -199,18 +66,22 @@ public function filter(mixed $value): mixed
         }
         $value = (string) $value;
 
-        $filtered = htmlentities($value, $this->getQuoteStyle(), $this->getEncoding(), $this->getDoubleQuote());
+        $filtered = htmlentities($value, $this->quoteStyle, $this->encoding, $this->doubleQuote);
         if (strlen($value) && ! strlen($filtered)) {
```

Comment:
Can you please tighten up the conditionals here:

```suggestion
        if (strlen($value) > 0 && strlen($filtered) === 0) {
```